WISE-Community/WISE
# Updated UI for milestone reports: split report content and recommendations, added tabs

Milestone reports now support two report template sections: report `content` and teacher `recommendations`. The report now shows the `content` in a section called "Class Report" and the `recommendations` in a section called "Recommendations". (See `"milestones"` in the `project.json` for the unit linked below to view the updated template json format.)

Report layout has also been simplified: Class Report is shown as the main section and now there are two tabs below that allow users to toggled between viewing Recommendations and Student Progress. (If no recommendations are specified in the template json, the Recommendations tab doesn't appear.)

To test:
1. Set up a run with with this unit: [61.zip](https://github.com/WISE-Community/WISE/files/3799392/61.zip)
2. As a student/students, complete some work on steps 2.4 and 2.5.
3. View the milestone reports in the Classroom Monitor as a teacher and verify that the new tabbed layout appears.

Closes #2052.
